frontend/src/components/DocumentViewer/DocumentViewBox.tsx

```diff
@@ -59,16 +59,18 @@ function DocumentViewBox({ closeOverlay }: { closeOverlay: () => void }) {
 					// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
 					tabIndex={0}
 				>
-					<DocViewer
-						documents={documentMetas}
-						activeDocument={documentMetas[documentIndex]}
-						pluginRenderers={DocViewerRenderers}
-						config={{
-							header: {
-								disableHeader: true,
-							},
-						}}
-					/>
+					<Suspense fallback={<p>loading...</p>}>
```

Comment:
Presumably you don't need this if you're no longer lazy loading anything in this component.